### PR TITLE
[BUG] Both (Extra items) Gloves and Visor not working + errors #135 + Added Bagoff as an Option under Radial menu as well

### PR DESCRIPTION
**Pull request by Bongmasta406**
The visor and gloves are not working correctly because they call in Id and title were capitalized... so i fixed it

**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]
#135

### CODE REVIEW

**Comment by Irishstevie** [comment]
#135

File: config.lua [hunk 1]
```diff
@@ -240,8 +240,8 @@ Config.MenuItems = {
                                 event = 'qb-radialmenu:ToggleProps',
                                 shouldClose = true
                             }, {
-                                id = 'Visor',
-                                title = 'Visor',
+                                id = 'visor',
+                                title = 'visor',
```
Comment (charj) [inline config.lua:244]: Would title not still need to be `Visor`?

File: config.lua [hunk 2]
```diff
@@ -282,8 +289,8 @@ Config.MenuItems = {
                                 event = 'qb-radialmenu:ToggleProps',
                                 shouldClose = true
                             }, {
-                                id = 'Gloves',
-                                title = 'Gloves',
+                                id = 'gloves',
+                                title = 'gloves',
```
Comment (charj) [inline config.lua:293]: As above

**Comment by charj** [comment]
A better title for this PR might be helpful to properly understand what the issue is here.

Also, please bump minor version number in `_maniest.lua`

**Pull request closed without merging**